### PR TITLE
Add support for resolving imported classes

### PR DESCRIPTION
**Pull request by kangax**

### CODE REVIEW

**Comment by kangax** [comment]
@atfzl I might have forgotten about some cases; take a look, let me know.

File: lib/core/index.js [hunk 1]
```diff
@@ -25,7 +26,8 @@ function dashesCamelCase (str: string) {
 }
 
 export const getFilePath = (context, styleFilePath) => {
-  const settings = context.settings['css-modules'];
+
+  const settings = context.settings && context.settings['css-modules'];
```
Comment (kangax) [inline lib/core/index.js]: unrelated to this PR but just makes it fail-safe if settings doesn't exist in .eslintrc config

**Comment by atfzl** [comment]
@kangax this change is causing issues as discussed in #44

**Comment by kangax** [comment]
@atfzl I'll take a look today and make a new PR

**Pull request merged**